scalafix-interfaces/src/main/java/scalafix/internal/interfaces/ScalafixCoursier.java

```diff
@@ -42,11 +42,23 @@
             String scalafixVersion,
             String scalaVersion
     ) throws ScalafixException {
-        Dependency scalafixCli = Dependency.parse(
+        List<Dependency> dependencies = new ArrayList<Dependency>();
+        dependencies.add(
+            Dependency.parse(
                 "ch.epfl.scala:::scalafix-cli:" + scalafixVersion,
                 ScalaVersion.of(scalaVersion)
+            ).withConfiguration("runtime")
         );
-        return fetch(repositories, Collections.singletonList(scalafixCli), ResolutionParams.create());
+        // Coursier does not seem to fetch runtime dependencies transitively, despite what Maven dictates
```

Comment:
I will follow up in coursier/coursier with a repro

Reply:
Actually, it works as expected when using sonatype:
```
$ cs resolve -r sonatype:snapshots -r central --no-default   ch.epfl.scala:scalafix-cli_2.13.8:0.9.34+25-b98e220d-SNAPSHOT:runtime | grep com.lihaoyi:pprint
com.lihaoyi:pprint_2.13:0.6.6:runtime
```

There is however something fishy with ivy2local: https://github.com/coursier/coursier/issues/2370.

So the code added there is probably needed only for tests to pass, but it does not hurt anyway.